### ISSUE TITLE
Re-running model this evening, estimates for NY have dramatically increased.

### ISSUE THREAD

**Issue by birdsarah** [jwrichar/COVID19-mortality#7]
Re-running the model this evening, compared to this morning, the estimates for NY have dramatically increased.

This morning:
![image](https://user-images.githubusercontent.com/1796208/77216510-c6e3db80-6ae8-11ea-94e7-4010ed365d89.png)

This evening:
![image](https://user-images.githubusercontent.com/1796208/77216529-ed097b80-6ae8-11ea-999a-92a8c822c4cf.png)

Any thoughts?

**Comment by jwrichar** [comment]
My read on it is that the MCMC sampler hasn't converged properly in the second plot.

I massively reduced the number of samples (and only 1 chain) for the live dashboard (because they needed everything to run in ~5 minutes).  If you increase the chain length (and tuning samples) you'll (hopefully) see a more stable model from run to run.

Also, I've noticed a fair amount of degeneracy in some of the parameters -- basically there are multiple local optima that produce ~fairly similar curves/results.  One thing you may try (I haven't gotten around to it yet) is fixing `alpha` to a large number (>=10) and running the MCMC sampler -- I'd expect this to give more stable results **and** can be interpreted as an assumption that South Korea's testing is ~complete (missing case rate of <1%).

**Comment by birdsarah** [comment]
Cool. I understand about half of this :). I'll try messing with those parameters.

**Comment by birdsarah** [comment]
I'm messing around with settings and parameters. For one run, I got the following. I think this means that 3 chains were good and one was erroneous. What would you recommend in this situation? 

![image](https://user-images.githubusercontent.com/1796208/77239153-4a630280-6ba5-11ea-90c0-57dd38775b01.png)

**Comment by birdsarah** [comment]
Another question. By fixing alpha, do you literally mean making it a constant e.g. `alpha=10`. If I do that then I basically get no error bars and the estimated and observed are identical.

**Comment by wcbeard** [comment]
@birdsarah my guess is he's referring to [this alpha](https://github.com/birdsarah/COVID19-mortality/blob/master/source/model.py#L107) (which might be what you tried). Looks like it's a bounded normal distribution, so maybe just bump up the lower bound, and increase `mu`? Like

```py
alpha = pm.Bound(pm.Normal, lower=10.0)('alpha', mu=15, sigma=3, shape=1)
```

I think that would move the prior for `tau_logit` closer to what you see in the traces above.

**Comment by jwrichar** [comment]
> I'm messing around with settings and parameters. For one run, I got the following. I think this means that 3 chains were good and one was erroneous. What would you recommend in this situation?

Exactly, 2 chains have converged to the same posterior and 1 went off into crazy land.  The general rule of thumb here would be to increase the "burn-in" period (number of `tune` samples).  You could also play around with the other `pm.sample` parameters, but I'm not an expert on PyMC3.

**Comment by jwrichar** [comment]
> @birdsarah my guess is he's referring to [this alpha](https://github.com/birdsarah/COVID19-mortality/blob/master/source/model.py#L107) (which might be what you tried).

Yes, exactly.

I literally mean setting it to a large value (interpretation here is that at the intercept -- normalized to South Korea's testing volume -- we deem that testing is "complete").  That would basically be achieved by doing:
```
        mu_model_logit = 100 + beta * people_per_test
        tau_logit = pm.Normal('tau_logit',
                              mu=mu_model_logit,
                              sigma=sigma,
                              shape=n)
```
So you still have all of the uncertainty / error bar estimates from the other parameters, we're simply taking one parameter out of the model.

**Comment by birdsarah** [comment]
Yes, that's what I did @jwrichar and the result was almost no uncertainty.

**Comment by jwrichar** [comment]
Interesting.  Not sure I'd expect that.

I've noticed a bit more instability in the curve estimation as the testing rates go up across the board -- I'm guessing that because there's less support at the low end (all states are above ~100 tests per million now, and most over 1000).

I also think a glaring weakness in the model of not accounting for population density is now coming into play, since the NY & NJ mortality rates are so high compared to average (the virus spreads faster than the hospital system can effectively handle).
